incubator/command-config/src/test/resources/io/aklivity/zilla/runtime/command/config/internal/asyncapi/http/proxy/tls/zilla.yaml

```diff
@@ -0,0 +1,83 @@
+name: example
+bindings:
+  tcp_server0:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port: 9090
+    routes:
+    - exit: tls_server0
+      when:
+      - port: 9090
+  tls_server0:
+    vault: server
+    type: tls
+    kind: server
+    options:
+      keys:
+      - "${{env.TLS_SERVER_KEYS}}"
```

Comment:
Perhaps this should be singular, `TLS_SERVER_KEY` as it is one key only in the environment variable?

Reply:
I changed it to singular everywhere. I am not sure why it was plural in the first place, probably just a typo.